demisto_sdk/commands/common/tests/integration_test.py

```diff
@@ -409,3 +410,52 @@ def test_invalid_feed(self, feed, fromversion):
         structure = mock_structure("", current)
         validator = IntegrationValidator(structure)
         assert not validator.is_valid_feed()
+
+    def test_is_fetch_params_exist_valid(self):
+        structure = mock_structure("")
+        validator = IntegrationValidator(structure)
+        validator.file_path = 'demisto_sdk/tests/test_files/Akamai_SIEM.yml'
+        with open(validator.file_path, 'r') as f:
+            _file = safe_load(f)
+        validator.current_file = _file
+        assert validator.is_fetch_params_exist(), 'is_fetch_params_exist() returns False instead True'
+
+    def _test_is_fetch_params_exist_sanity(self):
+        # missing param in configuration
+        structure = mock_structure("")
+        validator = IntegrationValidator(structure)
+        validator.file_path = 'demisto_sdk/tests/test_files/Akamai_SIEM.yml'
+        with open(validator.file_path, 'r') as f:
+            _file = safe_load(f)
```

Comment:
also, load directly to the validator object
```suggestion
            validator.current_file = safe_load(f)
```